pom.xml

```diff
@@ -182,7 +182,7 @@
             <dependency>
                 <groupId>com.fasterxml.jackson</groupId>
                 <artifactId>jackson-bom</artifactId>
-                <version>2.13.4</version>
+                <version>2.13.4.20221013</version>
```

Comment:
I guess version should be **2.13.4.2**

Reply:
I don't see `2.13.4.2` for `jackson-bom` here: https://search.maven.org/artifact/com.fasterxml.jackson/jackson-bom. What am I missing?

Reply:
As explained in https://github.com/FasterXML/jackson-databind/issues/3590

"2.13.4.2 micro-patch (jackson-bom 2.13.4.20221013). (NOTE: 2.13.4.1/2.13.4.20221012 have an issue that affects Gradle users)"